microsoft-edge/webview2/release-notes.md

```diff
@@ -14,7 +14,9 @@ The WebView2 team updates the [WebView2 SDK](https://www.nuget.org/packages/Micr
 
 Generally, release notes apply across the supported platforms, which are listed in [WebView2 API Reference](webview2-api-reference.md).
 
-WebView2 bug fixes, such as the fixes listed below, are either Runtime-specific or SDK-specific.
+WebView2 changes may require an update to the Runtime, SDK, or both. Most new APIs require both Runtime and SDK updates. Starting with our release in February 2023, all bug fixes will be marked as **Runtime-only** meaning only the Runtime needs to be updated, **SDK-only** meaning only the SDK needs to be updated, and no label means both need to be updated.

```

Comment:
WebView2 changes may require an update to the Runtime, SDK, or both. Most new APIs require both Runtime and SDK updates. Starting with the February 2023 release, the update requirements for each bug fix will be indicated as follows:

| Indicator | Meaning |
|---|---|
| No label | Both the Runtime and the SDK need to be updated. |
| **Runtime-only** | Only the Runtime needs to be updated. |
| **SDK-only** | Only the SDK needs to be updated. |